source/exploring_catalog_and_datasets/01_looking_up_the_catalog/catalog.rst

```diff
@@ -1,79 +1,179 @@
-Looking up the catalog
+Navigating the catalog
 ======================
 
-OpenDataSoft lets you find datasets with a rich search based experience. The catalog is often accessible via the **Explore** or **Datasets** link on the header menu. It allows to filter through all the portal's datasets by keyword, publisher, date or theme, among others.
+The dataset catalog of an OpenDataSoft portal is accessible from the front office for all users allowed on the portal (e.g. for open portals it could be anyone wheareas for private portals it should be only authenticated users).
 
-.. image:: images/explore_all.png
-   :alt: Explore catalog
+The catalog, depending on the portal, can be the default landing page of that portal. However, if that portal has another page as default landing page (e.g. a homepage), a link to the catalog should be in the navigation bar at the very top of the front office. Since this navigation bar is fully customizable, the label of the link to access the catalog can be different from one portal to another (very often, "Data", "Catalog", "Explore" or "Datasets" used as labels).
```

Comment:
very often, "Data", "Catalog", "Explore" or "Datasets" *are* used as labels ?

Reply:
Maybe this paragraph could be simplified with something like : "On most portals, the catalog is the landing page. However, if the default landing page has been changed to a custom page, the link to the catalog can often be found in the navigation bar at the ..." and keep the rest?